### ISSUE TITLE
the minitouch and screenshot can still work after disconnecting the device

### ISSUE THREAD

**Issue by zhouzxiang** [openstf/stf#819]
Good day!

**In the process of using STF, I found a Bug  which is necessary.**
**After it is finished using the device, minitouch and screenshots can still work, which is not common sense**

Reproduce the steps:
1. When the device is out of time or click to stop using on the list page, the operation page will pop up a box indicating that the device has finished using ,like this:
![capture_201836_103046](https://user-images.githubusercontent.com/23167666/37011229-2a5179c6-212a-11e8-9ee9-6f27f0f253cd.jpg)

2. Click on the external area of the box, the box will disappear immediately, the page still remain in the operation page.
![capture_201836_103222](https://user-images.githubusercontent.com/23167666/37011281-63b760fe-212a-11e8-916b-b4b894c7ad85.jpg)
![capture_201836_103304](https://user-images.githubusercontent.com/23167666/37011290-6934d6c4-212a-11e8-9f9d-d8e1ff0a2fec.jpg)

3. Now operating the device display area (although there is no image display), the actual device is still responding to the minitouch operation, in addition, the screenshot button can still correctly capture screenshots.
![capture_201836_103443](https://user-images.githubusercontent.com/23167666/37011414-fe6e6b6a-212a-11e8-8e38-ff99d115b974.jpg)

4.  I think this is not normal. Thank you for help!

**Comment by sorccu** [comment]
That’s a known issue. It wasn’t really a big deal for us so we never did anything about it. If someome submits a PR perhaps we can take a look at it.

**Comment by sorccu** [comment]
Good report by the way.

**Comment by zhouzxiang** [comment]
@sorccu Thx, @thinkhy says he will recently submit a PR to fix the issue.

**Comment by thinkhy** [comment]
@sorccu @zhouzxiang I think this would be a security vulnerabilities if we open services of STF to the public. So I will try to fix it.

**Comment by sorccu** [comment]
Yeah, but that’s not what we built the system for. Obviously I have nothing against a fix, though.
